src/components/scenes/Loans/LoanManageScene.tsx

```diff
@@ -122,7 +122,8 @@ export const LoanManageScene = (props: Props) => {
     [account, borrowEnginePluginId]
   )
   const excludeWalletIds = Object.keys(wallets).filter(walletId => walletId !== borrowEngineWallet.id)
-  const hardAllowedCollateralAsset = [{ pluginId: borrowEnginePluginId, tokenId: hardCollateralAddr }, { pluginId: 'bitcoin' }]
+  const hardAllowedCollateralAssets = [{ pluginId: borrowEnginePluginId, tokenId: hardCollateralAddr }]
+  if (actionOpType === 'loan-deposit') hardAllowedCollateralAssets.push({ pluginId: 'bitcoin', tokenId: undefined })
```

Comment:
Rename actionOpType to `userAction` and the type for this value `LoanManageUserAction` as an extra commit.